2020/8xxx/CVE-2020-8908.json

```diff
@@ -43,7 +43,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "A temp directory creation vulnerability exist in Guava versions prior to 30.0 allowing an attacker with access to the machine to potentially access data in a temporary directory created by the Guava com.google.common.io.Files.createTempDir(). The permissions granted to the directory created default to the standard unix-like /tmp ones, leaving the files open. We recommend updating Guava to version 30.0 or later, or update to Java 7 or later, or to explicitly change the permissions after the creation of the directory if neither are possible."
+		"value": "A temp directory creation vulnerability exists in all versions of Guava, allowing an attacker with access to the machine to potentially access data in a temporary directory created by the Guava API com.google.common.io.Files.createTempDir(). By default, on unix-like systems, the created directory is world-readable (readable by an attacker with access to the system). The method in question has been marked @Deprecated in versions 30.0 and later and should not be used. For Android developers, we recommend choosing a temporary directory API provided by Android, such as context.getCacheDir(). For other Java developers, we recommend migrating to the Java 7 API java.nio.file.Files.createTempDirectory() which explicitly configures permissions of 700, or configuring the Java runtime's java.io.tmpdir system property to point to a location whose permissions are appropriately configured."
```

Comment:
Are all versions of Guava affected or only versions > 9.09.15 as stated in the `version` field? Please correct the discrepancy.

Reply:
Version 9.09.15 was the first publicly available version of Guava and it effects everything after that.
What would be advisable here with the version value?

Reply:
Vuln reporter here: There are no fixed versions of Guava.